apps/ledger-live-mobile/src/components/DeviceAction/rendering.tsx

```diff
@@ -563,6 +568,71 @@ export function renderError({
   );
 }
 
+export function renderDeviceNotOnboarded({
+  t,
+  device,
+  navigation,
+}: {
+  t: TFunction;
+  device: Device;
+  // TODO: correctly type the navigation prop here AND in the DeviceAction component
+  navigation: any;
```

Comment:
Please do not use `any`.

Reply:
@alexandremgo you're currently typing the navigators, is that correct? this is to have return type of `useNavigation` in the device action